### ISSUE TITLE
Feature request: include keystore labels in animated QR exports

### ISSUE THREAD

**Issue by seedhammer** [sparrowwallet/sparrow#911]
The [BCR-2020-007](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-007-hdkey.md#cddl-for-hdkey) format for `crypto-hdkey` includes an optional `name` value:

```
	? name: text,                        ; A short name for this key.
```

We'd like to include keystore labels in our steel engravings, for which it would be nice if Sparrow exported them. I suppose the same information could be included in  PDF backups, so labels are restored from backups.

**Comment by craigraw** [comment]
As of 4bec71e7, Sparrow will now read the `name` value from scanned `ur:crypto-hdkey`s and set the keystore label if present. Sparrow does not currently show xpubs as `ur:crypto-hdkey` QRs however due to the need for wide compatibility.

Using this field for keystore labels in `ur:crypto-output` is a good idea too, so keeping this open for now.

**Comment by seedhammer** [comment]
Thank you. Indeed I wasn't clear: the feature request is about amending the `ur:crypto-output` QRs, not individual xpub QRs (which we don't use).

**Comment by craigraw** [comment]
Implemented in 76dc2947. Sparrow will now export keystore labels in displayed `ur:crypto-output` QRs (including PDF exports), and import keystore labels if present in the `ur:crypto-hdkey` components of scanned `ur:crypto-output` QRs. Note that Sparrow restricts keystore labels to 16 characters, and ensures that they are unique.
